tests/database/odbc-w-enclave/README.md

```diff
@@ -0,0 +1,39 @@
+# Database Test Case (ODBC Attestation)
+
+## Description
+
+This is the test case for database connection and query from sgxlkl application to a remote SQLServer database with always encrypted V2. After connection with attestation established, the test will do following queries and operations:
+
+1. before start, drop the test data table, CEK(Column Encrypted Key), CMK(Column Master Key)
+2. Create CMK (Column Master Key)
+3. create CEK (Column Encryption Key)
+4. Create Data table
+5. Insert Data into the table
+6. Query data
```

Comment:
We should drop the table as the final step, as other pipeline jobs don't know the exact table name anymore with the random suffix.